caps/eth.md

```diff
@@ -108,22 +108,173 @@ relayed by the peer.
 Transactions must be validated before re-propagating them. Relaying an invalid transaction
 results in peer disconnection.
 
+### Transaction Encoding and Validity
+
+Transaction objects exchanged by peers have one of two encodings, which are identical to
+the encodings used by Ethereum consensus. In definitions across this specification, we
```

Comment:
> which are identical to the encodings used by Ethereum consensus
I believe this is incorrect, typed txs exchanged over devp2p are rlp byte arrays, but the consensus encoding is not RLP.